assets/src/edit-story/output/utils/styles.js

```diff
@@ -67,6 +67,14 @@ function CustomStyles() {
                 }
               }
 
+              @media not all and (min-resolution:.001dpcm) {
+                @media {
+                  p.text-wrapper > span:first-of-type {
```

Comment:
Is `:first-of-type` necessary? Is there more than one?

Reply:
Yes, there can be multiple, inline styling can use multiple spans for example.

Reply:
Are those multiple spans direct children of the `<p>`? Asking since we're using the `>` combinator.

Reply:
Not really, now everything is wrapped to a single `span`. I mainly added this selector to ensure that only the required `span` gets the style to be extra sure since we can't really undo this change for the past stories -- it will be saved to post content statically.

I'll just remove it then since the way content is saved currently, it's not necessary.